src/coreclr/jit/importer.cpp

```diff
@@ -20611,16 +20608,6 @@ bool Compiler::IsTargetIntrinsic(NamedIntrinsic intrinsicName)
         case NI_System_Math_Round:
             return compOpportunisticallyDependsOn(InstructionSet_SSE41);
 
-        case NI_System_Math_FusedMultiplyAdd:
```

Comment:
Now that we are calling this (`isTargetIntrinsic`) as part of the early inlining observations, we might want to observe `Math.FMA` calls as target intrinsics, right?

Reply:
Indeed, thanks!